burr/core/action.py

```diff
@@ -239,6 +239,15 @@ def __repr__(self):
         write_repr = ", ".join(self.writes) if self.writes else "{}"
         return f"{self.name}: {read_repr} -> {write_repr}"
 
+    @property
+    def tags(self) -> list[str]:
+        """Returns the tags associated with this action.
+        Tags are effectively action aliases -- names that apply towards multiple actions.
```

Comment:
can this be applied to edges too?

Reply:
on thinking more applying to edges is fraught - so should mention where this alias is to be used?